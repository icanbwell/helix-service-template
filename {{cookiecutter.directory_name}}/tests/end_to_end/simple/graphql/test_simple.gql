query getProviders {
    providers(
        query_id: "foo"
        query: "Jim"
    ) {
        total_count
        results {
            result_id
        }
    }
}
