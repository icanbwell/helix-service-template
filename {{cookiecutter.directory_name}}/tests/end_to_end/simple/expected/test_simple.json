{
  "data": {
    "providers": {
      "results": [
        {
          "content": "John Madison",
          "content_auto": "John Madison"
        },
        {
          "content": "John McKay",
          "content_auto": "John McKay"
        }
      ],
      "total_count": 6
    }
  }
}
